tests/integration/components/event-table-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('event-table', 'Integration | Component | event table', {
```

Comment:
Please see other component tests and follow what is done here. Use the custom `moduleForComponent` helper.